addons/mail/static/src/core/messaging_service.js

```diff
@@ -80,7 +80,7 @@ export class Messaging {
      * Import data received from init_messaging
      */
     initialize() {
-        this.rpc("/mail/init_messaging", {}, { silent: true }).then(
+        this.rpc("/mail/init_messaging", { context: this.env.services.user.context }, { silent: true }).then(
             this.initMessagingCallback.bind(this)
         );
```

Comment:
According to the linting of the FW JS, we should write those lines like that. 🙂 

```suggestion
        this.rpc(
            "/mail/init_messaging",
            { context: this.env.services.user.context },
            { silent: true }
        ).then(this.initMessagingCallback.bind(this));
```